Ports/backward-cpp/patches/0002-backward-Pretend-to-be-Linux-with-some-modifications.patch

```diff
@@ -58,12 +58,13 @@ index ac7ad51738f8d13f07a6162d421b191696c9f84a..03f4f7546c4e59f3b17529c61a0f387e
  #else
  #warning ":/ sorry, ain't know no nothing none not of your architecture!"
  #endif
-@@ -4245,7 +4251,7 @@ public:
-     printer.address = true;
+@@ -4277,7 +4283,8 @@ public:
      printer.print(st, stderr);
  
--#if _XOPEN_SOURCE >= 700 || _POSIX_C_SOURCE >= 200809L
-+#if (_XOPEN_SOURCE >= 700 || _POSIX_C_SOURCE >= 200809L) && !defined(__serenity__)
+ #if (defined(_XOPEN_SOURCE) && _XOPEN_SOURCE >= 700) || \
+-    (defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE >= 200809L)
++    (defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE >= 200809L) &&
```

Comment:
This doesn't build for me. Did you try to do a ./package.sh clean && ./package.sh ?

```
[backward-cpp/build] [ 21%] Building CXX object CMakeFiles/test_main.dir/test/_test_main.cpp.o
[backward-cpp/build] In file included from .../git/serenity/Build/x86_64/Ports/backward-cpp/bombela-backward-cpp-65a769f/backward.cpp:36:
[backward-cpp/build] .../git/serenity/Build/x86_64/Ports/backward-cpp/bombela-backward-cpp-65a769f/backward.hpp:4286:64: error: operator '&&' has no right operand
[backward-cpp/build]  4286 |     (defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE >= 200809L) &&
[backward-cpp/build]       |                                                                ^
[backward-cpp/build] In file included from .../git/serenity/Build/x86_64/Ports/backward-cpp/bombela-backward-cpp-65a769f/backward.cpp:36:
[backward-cpp/build] .../git/serenity/Build/x86_64/Ports/backward-cpp/bombela-backward-cpp-65a769f/backward.hpp:4286:64: error: operator '&&' has no right operand
[backward-cpp/build]  4286 |     (defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE >= 200809L) &&
[backward-cpp/build]       |                                                                ^
```

Reply:
Oh, I messed up a rebase or something, I was supposed to have already fixed it.
I pushed a patch but didn't test it yet, I'm in the middle of a rebuild

Reply:
Looks like it did the trick!